app/models/prescription_details.rb

```diff
@@ -2,26 +2,26 @@
 
 class PrescriptionDetails < Prescription
   attribute :cmop_division_phone, String
-  attribute :in_cerner_transition, Boolean
+  attribute :in_cerner_transition, Boolean, sortable: { order: 'ASC' }
```

Comment:
I'm curious what changes this makes to the app's behavior. Are there any specs we can or should update to ensure we're changing what we want to?

Reply:
@robertbylight sorry to be thick, but I don't really understand what these changes will do, and whether they will affect the mobile app's use of the Rx api. Could you fill in a bit? Also as a general rule, we really prefer for tests to go in with the code they cover rather than in follow-on issues, particularly  when code in use by multiple teams is affected. (If that's the case here)

Reply:
yeah no problem, this details model is not used by mobile at the moment the only call to this model is used by the v1 module rx controller which is only being used by the vets-website. The changes here make it so that the attributes are able to be used to sort a collection. without this sortable piece if I use this attribute in a query param then the call would return an error.

Reply:
I agree the test should go with the pr, if its a show stopper for the approval then I can write it. I just decided not to spend time on it since this functionality is time sensitive for my team to get to the staging review.

Reply:
understood about the time sensitivity, given that and that mobile is not affected I'll approve.